noaa/challenges/sql.json

```diff
@@ -0,0 +1,275 @@
+    {% set task_options_10qps %}
+      "target-throughput": 10,
+      "clients": 2,
+      "warmup-iterations": 50,
+      "iterations": 100,
+      "tags": ["queries"]
+    {% endset %}
+
+    {% set task_options_5qps %}
+    "target-throughput": 5,
+    "clients": 2,
+    "warmup-iterations": 50,
+    "iterations": 100,
+    "tags": ["queries"]
+    {% endset %}
+
+    {% set task_options_1qps %}
+    "target-throughput": 1,
+    "clients": 2,
+    "warmup-iterations": 25,
+    "iterations": 50,
+    "tags": ["queries"]
+    {% endset %}
+
+    {
+      "name": "sql",
+      "description": "Checks the performance of SQL",
+      "default": false,
+      "schedule": [
+        {
+          "operation": "delete-index",
+          "tags": ["setup"]
+        },
+        {
+          "operation": {
+            "operation-type": "create-index",
+            "settings": {{index_settings | default({}) | tojson}}
+          },
+          "tags": ["setup"]
+        },
+        {
+          "name": "check-cluster-health",
+          "operation": {
+            "operation-type": "cluster-health",
+            "index": "weather-data-2016",
+            "request-params": {
+              "wait_for_status": "{{cluster_health | default('green')}}",
+              "wait_for_no_relocating_shards": "true"
+            },
+            "retry-until-success": true
+          },
+          "tags": ["setup"]
+        },
+        {
+          "operation": "index",
+          "#COMMENT": "This is an incredibly short warmup time period but it is necessary to get also measurement samples. As this benchmark is rather about search than indexing this is ok.",
+          "warmup-time-period": 10,
+          "clients": {{bulk_indexing_clients | default(8)}},
+          "ignore-response-error-level": "{{error_level | default('non-fatal')}}",
+          "tags": ["setup"]
+        },
+        {
+          "name": "refresh-after-index",
+          "operation": "refresh",
+          "clients": 1,
```

Comment:
You can specify `clients` but the default value is `1` anyway so you might just as well rely on the default.